jbpm-services/jbpm-services-api/src/main/java/org/jbpm/services/api/model/UserTaskInstanceWithPotOwnerDesc.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.services.api.model;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+public interface UserTaskInstanceWithPotOwnerDesc {
```

Comment:
why not extending UserTaskInstanceDesc and add only additional methods?

Reply:
Done